app/views/home/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<main class="home--main">
+  <div></div>
+  <h1>Expense Insight</h1>
+  <div class="home--link--container">
+    <%= link_to 'LOG IN', new_user_session_path %>
+    <%= link_to 'SIGN UP', new_user_registration_path %>
+  </div>
+</main>
```

Comment:
- [ ] As required for the project your app to follow the the [the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) from Behance, kindliy ensure that you use the typography(font, size, weight,...), text-color, etc.

| Template design | your Design | 
| ---------------------- | -------------------- |
| ![Screenshot from 2023-09-22 20-53-11](https://github.com/Akorede3133/expense-insight/assets/83042742/d759bde4-9bb7-427f-ad6e-669102f1b92b) | ![image](https://github.com/Akorede3133/expense-insight/assets/83042742/476561e5-0baa-4cd4-bb63-c8908d306c8c) |